packages/@o3r/testing/schematics/ng-add/index.ts

```diff
@@ -120,7 +114,7 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
           const jestConfigFilesForProject = () => mergeWith(apply(url('./templates/project'), [
             template({
               ...options,
-              rootRelativePath: getRelativePath(workingDirectory)
+              rootRelativePath: path.relative(workingDirectory, '.' + tree.root.path)
```

Comment:
```suggestion
              rootRelativePath: path.relative(workingDirectory, tree.root.path.replace(/^\//, './'))
```

Would be safer in case it does not start with `/`.